regulation/node.py

```diff
@@ -73,7 +73,7 @@ def to_json(self):
         node_dict['text'] = self.text
         if self.title and self.title != '':
             node_dict['title'] = self.title
-        if self.marker:
+        if self.marker is not None:
```

Comment:
I will also say, this is intended to [match expectations in reg-site](https://github.com/cfpb/regulations-site/blob/master/regulations/templates/regulations/tree-with-wrapper.html#L6), where only if `marker` is `""` or `"none"` does it get a "markerless" attribute. If the `marker` doesn't exist, then reg-site punts.

This is partially because our old data doesn't have a `marker` attribute for any nodes, marked or markerless.